encrypted-media-respec.html

```diff
@@ -3179,44 +3193,66 @@
               <p class="note">For example, <a def-id="temporary-session"></a> and <a def-id="persistent-license-session"></a> sessions may have different requirements.</p>
             </li>
             <li>
-              <p>Implementations that <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">use Distinctive Identifier(s) or Distinctive Permanent Identifier(s)</a> SHOULD support the option to not use them.
-                When supported, applications can select for this mode using <a def-id="option-distinctiveIdentifier"></a> = <a def-id="requirement-not-allowed"></a>.
-                Selecting such an option MAY affect the results of the <a def-id="requestMediaKeySystemAccess"></a> call and/or the license requests that are generated from subsequently generated sessions.
+              <p>
+                Implementations that <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">use Distinctive Identifier(s) or Distinctive Permanent Identifier(s)</a> SHOULD support the option to not use them.
+                Implementations with such support SHOULD expose the ability for the user to select this option.
```

Comment:
I think this specific line is new. That's fine - I'll open a GitHub issue. I'm wondering whether it would be better to include this text and add an ISSUE box that it needs discussion or remove it and assume it is not substantive and could be added. Thoughts?